www/phantomjs/Portfile

```diff
@@ -1,17 +1,18 @@
 # -*- coding:utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           bitbucket 1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
 
-bitbucket.setup     ariya phantomjs 2.0.0
-revision            3
+# please update version of phantomjs-qt-qtbase and phantomjs-qt-qtwebkit
+github.setup        ariya phantomjs 2.1.1
```

Comment:
`phantomjs` development is suspended: https://github.com/ariya/phantomjs/issues/15344

Suggestion: Add a `notes` string that alerts users to this reality, and suggest that they use port `chromedriver` instead.

Reply:
Thank you, that is a good idea.
I will get back to this pull request in a a couple of day.